src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dayRent = 40;
+  let totalRental = 0;
+
+  if (days >= 7) {
+    totalRental = dayRent * days - 50;
+  } else if (days >= 3) {
+    totalRental = dayRent * days - 20;
+  } else {
+    totalRental = dayRent * days;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions. Also, don't add `else` after `if` with `return`. See checklist items 4 and 5 for reference.